docs/src/quickstart.md

```diff
@@ -0,0 +1,125 @@
+# Quickstart!
+
+This page serves as a quickstart guide and it will get you up and running with whatever you need to start documenting YourAwesomePackage.jl . Do refer to the rest of the documentation if you want to make use of the entire feature set :)
+
+## What are we trying to do?
+
+This
+
+![](assets/docu.png)
+
+## How 
+
+1. Toml who? This is very generic to Julia and you can get it by going to your folder, opening Julia and typing these. [Source](https://invenia.github.io/PkgTemplates.jl/stable/user/)
+
+    ```jl
+    ]add PkgTemplates
+    using PkgTemplates
+    t = Template()
+    t("MyPkg")    
+    ```
+This will create a bunch of default folders which we will use.
+> Note that PkgTemplates can [generate the documentation for you](https://invenia.github.io/PkgTemplates.jl/stable/user/#PkgTemplates.Documenter). It requires a bit of tinkering but you can skip a few steps if you want. (Go to step 8 directly :) )
+2. Get the package
+   ```jl
+   ]add Documenter
+   ```
+3. Put your code in the folder called src
+4. Create a folder called docs
+    - Follow this structure if you can (most of this is auto generated)
+      - DDL.jl
+        - src
+          - DDL.jl
+          - classif.jl
+        - docs
+        - tests
+        - README.md
+        - Manifest.toml
+        - Project.toml
```

Comment:
Put this directory structure in a code block rather, since it's nested in a list already the additional nesting doesn't make it stand out enough.